cmd/manager/main.go

```diff
@@ -3,10 +3,10 @@ package main
 import (
```

Comment:
Given that this work is now tracked via a bug, we will need to rewrite the commit message and change the PR header.